packages/dx-react-chart-demos/src/demo-sources/layout/material-ui/legend.jsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  BarSeries,
+  Legend,
+  Title,
+} from '@devexpress/dx-react-chart-material-ui';
+import { withStyles } from '@material-ui/core/styles';
+
+import { Stack } from '@devexpress/dx-react-chart';
+
+import { ageStructure } from '../../../demo-data/data-vizualization';
+
+const styles = {
+  root: {
+    width: '60%',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  },
+  title: {
+    margin: '0px',
+  },
+  legendRoot: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  item: {
+    flexDirection: 'column',
+    margin: 'auto',
+  },
+  label: {
+    margin: '10px',
+  },
+};
+
+const RootWithTitle = withStyles(styles)(({ classes, ...restProps }) => (
```

Comment:
When legend is positioned at the top of the chart its title is indistinguishable from chart title. Looks like legend should be placed at the right (or left) side of chart.

Reply:
If we place the legend at the right or left side, we should think about how legend should looks. [Legend at the right](https://www.screencast.com/t/XwY5egaW) is not demonstrated legend layout customization, only title with the legend.  Another variant is [this](https://www.screencast.com/t/aDnh8JEo), but I don't think it looks good.

Reply:
Primary purpose is to show hybrid rendering.
While [this](https://www.screencast.com/t/XwY5egaW) does not contain layout customization code, [this](https://www.screencast.com/t/aDnh8JEo) looks too weird (at least for a demo) - I think we should stick with the first variant.

Reply:
[Now](https://www.screencast.com/t/EoGQW7io1j9x)

Reply:
Let's align legend items by the right side and align right sides of title and legend items. And let's decrease vertical gap between title and items.

![pr2005_2](https://user-images.githubusercontent.com/2501124/57230346-d701e200-7020-11e9-9f31-778b3a0bd4c8.png)